src/language/ast.js

```diff
@@ -242,3 +242,52 @@ export type NonNullType = {
   loc?: ?Location;
   type: NamedType | ListType;
 }
+
+export type TypeDefinition = {
```

Comment:
These should probably go into another file, clientAST.js or something like that.